src/Backend/AppCenter.vala

```diff
@@ -0,0 +1,63 @@
+// -*- Mode: vala; indent-tabs-mode: nil; tab-width: 4 -*-
+//
+//  Copyright (C) 2017 Slingshot Developers
```

Comment:
Can you assign copyright either to elementary LLC or yourself or both. I don't think "Slingshot Developers" is really a valid copyright holder since it's not really a legit organization